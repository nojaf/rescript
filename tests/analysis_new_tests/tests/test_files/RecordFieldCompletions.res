// == TEST: Record field completion in nested record
let x = TestTypeDefs.nestedTestRecord.
//                                    ^crm

// == TEST: Record field completion in nested record, another level
let x = TestTypeDefs.nestedTestRecord.nested.
//                                           ^crm