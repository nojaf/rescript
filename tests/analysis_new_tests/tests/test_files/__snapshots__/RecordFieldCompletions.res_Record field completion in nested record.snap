Complete /Users/nojaf/Projects/rescript/tests/analysis_new_tests/tests/test_files/.build/RecordFieldCompletions_1.res 1:38
Found Completable: Cexpression

1 + // Record field completion in nested record
2 + let x = TestTypeDefs.nestedTestRecord.
            ----------------------------- 
3 + //                                    ^com
4 + 

[{
    "label": "test",
    "kind": 5,
    "tags": [],
    "detail": "bool",
    "documentation": {"kind": "markdown", "value": "```rescript\ntest: bool\n```\n\n```rescript\ntype nestedTestRecord = {\n  test: bool,\n  nested: {name: string, oneMoreLevel: {here: bool}},\n}\n```"}
  }, {
    "label": "nested",
    "kind": 5,
    "tags": [],
    "detail": "\\\"nestedTestRecord.nested\"",
    "documentation": {"kind": "markdown", "value": "```rescript\nnested: \\\"nestedTestRecord.nested\"\n```\n\n```rescript\ntype nestedTestRecord = {\n  test: bool,\n  nested: {name: string, oneMoreLevel: {here: bool}},\n}\n```"}
  }]

